src/assets/styles/modules/responsive-images.scss

```diff
@@ -0,0 +1,59 @@
+/* ================ Responsive images ================ */
+.responsive-image__container {
+  position: relative;
+  overflow: hidden;
+  width: 100%;
+  margin: 0 auto;
+}
+
+.responsive-image__wrapper {
+  position: relative;
+  display: block;
+
+  img {
+    display: block;
+    width: 100%;
+  }
+}
+
+.responsive-image__image {
+  position: absolute;
+  top: 0;
+  left: 0;
+  display: block;
+  width: 100%;
+  height: 100%;
+  cursor: pointer;
+}
+
+/*================ Responsive background images ================*/
+.responsive-bg-image-wrapper {
+  overflow: hidden;
+  height: 278px;
```

Comment:
I took the code from Narrative and we need to set a minimum height, otherwise we would not see the background image. Do you want me to set a minimum height?